bower.json

```diff
@@ -6,10 +6,10 @@
     "qunit-phantom-runner": "jonkemp/qunit-phantomjs-runner#1.2.0"
   },
   "devDependencies": {
-    "backburner": "https://github.com/ebryn/backburner.js.git#f4bd6a2df221240ed36d140f0c53c036a7ecacad",
-    "rsvp": "https://github.com/tildeio/rsvp.js.git#3.0.14",
+    "backburner": "https://github.com/ebryn/backburner.js.git",
```

Comment:
@stefanpenner this is using `master` of backburner.

Should I peg it to a particular commit?